driver/registry_sql.go

```diff
@@ -135,32 +133,32 @@ func (m *RegistrySQL) SchemaMigrationPlan() (*tablewriter.Table, error) {
 	return table, nil
 }
 
-func (m *RegistrySQL) CreateSchemas() (int, error) {
+func (m *RegistrySQL) CreateSchemas(driver string) (int, error) {
```

Comment:
Maybe rename this to `dbName`, otherwise this would be considered the driver which would be expected to be `postgres` for CockroachDB.